scripts/dts/edtlib.py

```diff
@@ -550,6 +550,11 @@ def _init_prop(self, name, options):
             # 'category: optional' property that wasn't there
             return
 
+        enum = options.get("enum")
+        if enum and val not in enum:
+            _err("value ({}) for property ({}) is not in enumerated "
```

Comment:
"value '{}' for property '{}' is not..." maybe, to be consistent (and since the name is essential info).

Reply:
Done.